cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -73,8 +73,10 @@ def add_user_info(self, user_info):
                 'lastName': user_info['lastName'],
                 'Gender': user_info['Gender'],
                 'DOB': user_info['DOB'],
-                'Grad_date': user_info['Grad_Date'],
-                'Major': ', '.join(sorted(user_info['Major'])),
+                'Position': user_info['UserPosition'],
+                'GradSemester': user_info.get('GradSemester', ' '),
+                'GradYear': user_info.get('GradYear', ' '),
```

Comment:
Are these really optional? Should they be optional based upon "UserPosition"

Reply:
Yeah they should be optional based upon the UserPosition field, but I didn't think that was something that could be done. I can look into it right now though.